pandas/core/generic.py

```diff
@@ -3136,6 +3136,22 @@ def to_latex(
         if multirow is None:
             multirow = config.get_option("display.latex.multirow")
 
+        if is_list_like(formatters) and not isinstance(formatters, dict):
```

Comment:
can you add an Example that uses this (e.g. similar to your test is fine)

Reply:
I was having an issue providing an example with lambda functions in doc, so I included  a simpler example.